src/redux/actions/RatingsActions/ratingActions.test.js

```diff
@@ -0,0 +1,17 @@
+import * as types from './types';
+import * as actions from './actions';
+
+describe('test rating actions', () => {
+  it('should dispatch SELECT RATE', () => {
+    expect(actions.selectedRating({}).type).toBe(types.SELECT_RATE);
```

Comment:
Why are we just looking at the action type when testing?

Reply:
Hello @sharkdevs 

The steps you have given are working as expected which is good.
My only concern is how is this linked to an article so that the user actually rates an article?

Regards
Steven Ennis

Reply:
the rating shall take the article slug @Teatoller, once articles is merged, the component shall work with the rest of the articles.